vscode/microsoft-kiota/src/workspaceTreeProvider.ts

```diff
@@ -3,12 +3,16 @@ import * as path from 'path';
 import * as fs from 'fs';
 import { KIOTA_DIRECTORY, KIOTA_WORKSPACE_FILE } from './constants';
 
-const workspaceJsonPath = path.join(vscode.workspace.workspaceFolders?.map(folder => folder.uri.fsPath).join('') || '', KIOTA_DIRECTORY, KIOTA_WORKSPACE_FILE);
+export function getWorkspaceJsonPath(): string {
```

Comment:
this one and the other function in utils need to be merged

Reply:
yeah, I've replaced the other one wih this one